pkg/apis/serving/k8s_validation.go

```diff
@@ -109,6 +110,24 @@ func ValidateVolumes(ctx context.Context, vs []corev1.Volume, mountedVolumes set
 	return volumes, errs
 }
 
+func shouldAllowPersistentVolumeClaims(volume corev1.VolumeSource, features *config.Features) *apis.FieldError {
+	var errs *apis.FieldError
+	if volume.PersistentVolumeClaim == nil {
+		return nil
+	}
+	if features.PodSpecPersistentVolumeClaim != config.Enabled {
+		errs = errs.Also(&apis.FieldError{Message: fmt.Sprintf("Persistent volumes claims support is disabled, "+
+			"but found Persistent Volume Claim %s", volume.PersistentVolumeClaim.ClaimName)})
```

Comment:
Another nit pick, sorry ...

```suggestion
		errs = errs.Also(&apis.FieldError{Message: fmt.Sprintf("Persistent volume claim support is disabled, "+
			"but found persistent volume claim %s", volume.PersistentVolumeClaim.ClaimName)})
```

(the capitalization should be consistent, I would prefer to user lower case for 'persistent volume claim' and camel-case for `PersistenceVolumeClaim`)

Reply:
ok

Reply:
Btw `Persistent Volume Claim` refers to the K8s api attribute. `Persistent volumes claims support` refers to the feature of supporting this type of volumes.

Reply:
fixed

Reply:
> @rhuss Here you suggested the opposite no? I changed it due to that comment. I guess this is an update to that comment?

Yep, for me, it's important to be consistent (it doesn't make sense to reference the K8s API attribute (which btw IMO *is* [`PersistentVolumeClaim`](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.21/#persistentvolumeclaim-v1-core) and the feature in the error message (actually the _feature_ is disabled, not the API attribute).

Also (as @julz mentioned) `persistent volumes claims` sounds incorrect with the double plural (regardless of the arguments above)